.gitignore

```diff
@@ -2,3 +2,4 @@
 *.swp
 *.sublime-project
 *.sublime-workspace
+*~
```

Comment:
Oh, that was an "along the way". Tilde files are commonly dropped as backups by e.g. Emacs, and they're normal to ignore.